play.py

```diff
@@ -12,13 +12,15 @@ def play(self):
         self.ui.display("Welcome to Battleship")
         board_full = False
         ship_orientation = 'row'
-        self.place.ship_fit(self.board.state, self.ships.aircraft_carrier['size'], ship_orientation)
+        self.board.add_to_board(self.ships.all_ships, self.place, ship_orientation)
+
         while not board_full:
-            board_full = self.validate.board_full(self.board.state)
+            board_full = self.validate.board_full(self.board.state) 
```

Comment:
So it seems like this would catch the board being full the turn after it is full.